packages/exports/src/exportQuizDesignerSlides.ts

```diff
@@ -21,14 +22,10 @@ export const exportQuizDesignerSlides = async ({
   onStateChange: (state: State<OutputData>) => void;
 }): Promise<Result<OutputData>> => {
   try {
-    const speakerNotesTagsToDelete: SpeakerNotesTag[] = [];
-    // See how many questions there are
-    const numberOfQuestions = quiz.questions.length;
-    for (let i = 1; i <= 20; i++) {
-      if (i > numberOfQuestions) {
-        speakerNotesTagsToDelete.push(`question${i}`);
-      }
-    }
+    const speakerNotesTagsToDelete: SpeakerNotesTag[] = Array.from(
+      { length: 20 - quiz.questions.length },
+      (_, i) => i + quiz.questions.length + 1,
+    ).map((n) => `question${n}` as SpeakerNotesTag);
```

Comment:
Do we need the assertion here? Will annotation not do?

Reply:
For some reason TS complains unless you have the as. I think because it's a runtime string manipulation

Reply:
I think it could be that typescript can no longer infer the numbers coming from the Array used to build the strings. You should find that it works if you type Array.from or n to be `1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20`

Reply:
Okay I've done some extra work to tighten the types here even further